Authenticator/Source/RootViewController.swift

```diff
@@ -84,7 +84,10 @@ extension RootViewController: AppPresenter {
             dismissViewController()
 
         case .Scanner:
-            let scannerViewController = TokenScannerViewController(dispatchAction: dispatchAction)
+            let scannerViewController = TokenScannerViewController(dispatchAction:
```

Comment:
Opening Brace Spacing Violation: Opening braces should be preceded by a single space and on the same line as the declaration. (opening_brace)